2000-10-10  Jakub Jelinek  <jakub@redhat.com>

	* gcc.dg/991209-1.c (stack_ptr): Use __asm, not asm, so that -ansi
	does not error on it.

--- gcc/testsuite/gcc.dg/991209-1.c.jj	Fri Dec 10 01:57:03 1999
+++ gcc/testsuite/gcc.dg/991209-1.c	Tue Oct 10 17:34:06 2000
@@ -5,4 +5,4 @@ int foo ()
   return 1;
 }
 
-register char *stack_ptr asm ("%esp");
+register char *stack_ptr __asm ("%esp");
